goal_src/jak1/pc/subtitle.gc

```diff
@@ -787,10 +894,54 @@
       (none))
 
     :code (-> subtitle-process code)
-    :post (-> subtitle-process post)
+    :post (behavior ()
+      (setup-subtitle-font)
+      (draw-subtitle)
+      (draw-debug-text-box (-> self font))
+      0)
 
     )
 
+(defstate subtitle-debug-checking-lines (subtitle)
+
+    :trans (behavior () (setup-subtitle-font) (none))
+    :code (behavior ()
+      (protect ((-> *pc-settings* subtitle-speaker?))
+      (set! (-> *pc-settings* subtitle-speaker?) #t)
+      (let ((lines-so-far 0)
+            (lines-this-time 0)
+            (bad-lines 0))
+        (dotimes (i (length *subtitle-text*))
+          (dotimes (ii (length (-> *subtitle-text* data i)))
+            (when (= lines-this-time PC_SUB_DBG_CHECK_GROUP_SIZE)
+                (set! lines-this-time 0)
+                (suspend))
+
+            (set! (-> self want-subtitle) (-> *subtitle-text* data i keyframes ii))
+            (1+! lines-this-time)
+            (when (< (* (-> *SUBTITLE-bank* lines) 11) (print-game-subtitle (subtitle-format self (-> self want-subtitle)) (-> self font) #f 128 22))
+                (format 0  "LINE ~D IN SCENE ~D IS TOO LARGE!~%" (1+ ii) (1+ i))
+                (format #t "LINE ~D IN SCENE ~D IS TOO LARGE!~%" (1+ ii) (1+ i))
```

Comment:
Might be good to also add the spool/hint name here, so you don't have to manually scroll to the scene in the subtitle debug screen to find out which one it is.

`(format 0  "LINE ~D IN SCENE ~D (~S) IS TOO LARGE!~%" (1+ ii) (1+ i) (-> *subtitle-text* data i name))`